Content.Server/Chat/Managers/IChatManager.cs

```diff
@@ -12,15 +12,18 @@ public interface IChatManager
         /// <summary>
         ///     Dispatch a server announcement to every connected player.
         /// </summary>
-        void DispatchServerAnnouncement(string message);
+        /// <param name="message"></param>
+        /// <param name="colorOverride">Override the color of the message being sent.</param>
+        void DispatchServerAnnouncement(string message, Color colorOverride = default);
```

Comment:
Why is the default, well, `default`. Shouldn't it just be the actual default color.

Reply:
I only did it that way because the way the color is changed is by using "MessageColorOverride" so if no override is given, it just doesn't get called

Reply:
It's probably fine to just.. always override it. Also note the default for Color is a valid color value!

Reply:
Not that it would be a big issue in this case, since the change you are suggesting would mean we'd have to add the color for `NetMessageToAll` in the OOC, as both OOC and Radio use `NetMessageToAll` right now, which pulls the color from the channel itself. So providing `NetMessageToAll` an actual color for default would override the channel color

Reply:
Made colorOverride nullable